lckernel/cad/math/intersectionhandler.cpp

```diff
@@ -289,3 +289,66 @@ void Intersection::BezCir(const geo::Circle C, const geo::Area c_area, const geo
     BezCir(C, c_area, m_area, b2split[1], ret);
 
 }
+
+std::vector<geo::Coordinate> Intersection::SplineLine(geo::Spline B, const geo::Vector& V) {
+    std::vector<geo::Coordinate> ret;
+    auto beziers = B.beziers();
+    for(const auto & bezier : beziers) {
+        auto vecret = BezierLine(bezier, V);
```

Comment:
I have kept the names after thoughts. We preferred to avoid using get for getters. Just the names. Like instead of getRadius, just the radius(). Same way I wanted for intersections. and also about using the intersection word, like BezierBezierIntersection, I have already kept them in intersection namespace. so each of them is to be used as Intersection::BezierBezier or Intersection::LineBezier etc. If you still want I can change as required and for the same reason I actually kept the first characters uppercase as well. reading this please suggest me what should be done.